packages/gelly-gmod/src/source/GetCubemap.cpp

```diff
@@ -19,6 +19,8 @@ using namespace Gelly::DataTypes;
 
 using TextureHandle_t = ulonglong;
 
+// Clang only supports type trait function pointers, same with GCC
```

Comment:
Should this be `Only clang/gcc supports ...`, otherwise couldn't we use this block of code for MSVC compiles too?

Not a blocker for this PR though

Reply:
Yeah, thats good. I tested on both Clang and GCC and none of them support MSVC-style function declarations, they only accepted calling conventions via a type trait. On the other hand, MSVC does *not* accept type trait function declarations in which we go the old-school method (although we make it a little more modern by using `using`).